src/features/import/components/Configure/Preview/OrgPreview.tsx

```diff
@@ -0,0 +1,43 @@
+import { Theme } from '@mui/system';
+import { Typography } from '@mui/material';
+
+import messageIds from 'features/import/l10n/messageIds';
+import PreviewGrid from './PreviewGrid';
+import { ZetkinOrganization } from 'utils/types/zetkin';
+import { ColumnKind, Sheet } from 'features/import/utils/types';
+import { Msg, useMessages } from 'core/i18n';
+
+interface OrgPreviewProps {
+  currentSheet: Sheet;
+  theme: Theme;
+  org: Pick<ZetkinOrganization, 'title' | 'id'> | undefined;
+}
+
+const OrgPreview = ({ currentSheet, org, theme }: OrgPreviewProps) => {
+  const messages = useMessages(messageIds);
+
+  const hasMapped = currentSheet.columns.some(
+    (column) =>
+      column.kind === ColumnKind.ORGANIZATION && column.mapping.length > 0
+  );
+  const noOrg = (
+    <Typography
+      sx={{
+        color: theme.palette.grey[400],
+        fontStyle: 'italic',
+      }}
+    >
+      (<Msg id={messageIds.configuration.preview.noOrg} />)
+    </Typography>
```

Comment:
This pattern repeats for every type of preview. I would consider putting it inside of `PreviewGrid` instead, and just have the label be passed in as a property.